packages/components/src/select-control/control.js

```diff
@@ -114,6 +114,7 @@ class Control extends Component {
 	renderInput() {
 		const {
 			activeId,
+			isDisabled,
```

Comment:
Can this just be `disabled`?

Reply:
Good idea! I changed that in the commit 709b80b